mmpose/datasets/datasets/bottom_up/bottom_up_coco.py

```diff
@@ -204,7 +204,7 @@ def evaluate(self, outputs, res_folder, metric='mAP', **kwargs):
             num_keypoints: K
 
         Args:
-            outputs (list(preds, scores, image_path)):Output results.
+            outputs (list(preds, scores, image_path, output_cpu)):
```

Comment:
could `output_cpu` be more specific/concrete? say, things like `heatmap` etc.

Reply:
changed to `output_heatmap`